packages/currency/src/conversion-aggregators.ts

```diff
@@ -49,6 +49,7 @@ const noConversionNetworks: AggregatorsMap = {
   avalanche: {},
   bsc: {},
   optimism: {},
+  moonbeam: {},
```

Comment:
I made a proposal in https://github.com/RequestNetwork/requestNetwork/pull/1003 to remove this